### PR TITLE
Feat/condo/doma-7494/multipayment for invoice

### PR DESCRIPTION
**Pull request by AleX83Xpert**
This PR brings the functionality to create multipayments for invoices using the same mutation as for creating multipayment for receipts.

### CODE REVIEW

File: apps/condo/domains/acquiring/schema/MultiPayment.js [hunk 1]
```diff
@@ -56,8 +58,13 @@ const { RESIDENT, STAFF } = require('@condo/domains/user/constants/common')
 
 const { ACQUIRING_INTEGRATION_FIELD } = require('./fields/relations')
 
-
-
+const ERRORS = {
+    NOT_UNIQUE_INVOICES: {
+        code: BAD_USER_INPUT,
+        type: MULTIPAYMENT_NOT_UNIQUE_INVOICES,
+        message: 'Not unique invoices',
```
Comment (sitozzz) [inline apps/condo/domains/acquiring/schema/MultiPayment.js:65]: There are some other fields, that would be great to define here. Eg `messageForUser` and `fields`

File: apps/condo/domains/acquiring/schema/MultiPayment.js [hunk 2]
```diff
@@ -56,8 +58,13 @@ const { RESIDENT, STAFF } = require('@condo/domains/user/constants/common')
 
 const { ACQUIRING_INTEGRATION_FIELD } = require('./fields/relations')
 
-
-
+const ERRORS = {
+    NOT_UNIQUE_INVOICES: {
+        code: BAD_USER_INPUT,
+        type: MULTIPAYMENT_NOT_UNIQUE_INVOICES,
```
Comment (sitozzz) [inline apps/condo/domains/acquiring/schema/MultiPayment.js:64]: I can't spot test case for this error

Comment (AleX83Xpert) [reply apps/condo/domains/acquiring/schema/MultiPayment.js:64]: Just added the test.

File: apps/condo/domains/acquiring/schema/MultiPayment.js (hunk 1 above)
Comment (AleX83Xpert) [reply apps/condo/domains/acquiring/schema/MultiPayment.js:65]: This error is for API. Residents can't check the invoice twice within the interface.

**Review by sitozzz — APPROVED** [review]

File: apps/condo/domains/marketplace/schema/InvoiceContext.test.js [hunk 3]
```diff
@@ -32,26 +33,28 @@ const {
...
 
 let adminClient, supportClient, anonymousClient
+let integration
 
 describe('InvoiceContext', () => {
     beforeAll(async () => {
         adminClient = await makeLoggedInAdminClient()
         supportClient = await makeClientWithSupportUser()
         anonymousClient = await makeClient()
+        ;[integration] = await createTestAcquiringIntegration(supportClient)
```
Comment (ekabardinsky) [inline apps/condo/domains/marketplace/schema/InvoiceContext.test.js:43]: ; why it here?

**Review by ekabardinsky — APPROVED** [review]

File: apps/condo/domains/acquiring/schema/MultiPayment.js [hunk 4]
```diff
@@ -250,27 +257,48 @@ const MultiPayment = new GQLListSchema('MultiPayment', {
                 if (alreadyWithMPPayments.length) {
                     addValidationError(`${MULTIPAYMENT_PAYMENTS_ALREADY_WITH_MP} Failed ids: ${alreadyWithMPPayments.join(', ')}`)
                 }
+
+                const receiptPayments = payments.filter(({ invoice, receipt }) => !!receipt && !invoice)
```
Comment (dkoviazin) [inline apps/condo/domains/acquiring/schema/MultiPayment.js:261]:
```js
const { receiptPayments, invoicePayments, virtualPayments } = payments.reduce(
  (acc, { invoice, receipt }) => {
    if (!!receipt && !invoice) {
      acc.receiptPayments.push({ invoice, receipt })
    } else if (!!invoice && !receipt) {
      acc.invoicePayments.push({ invoice, receipt })
    } else if (!invoice && !receipt) {
      acc.virtualPayments.push({ invoice, receipt })
    }
    return acc
  },
  { receiptPayments: [], invoicePayments: [], virtualPayments: [] }
);
```
This code will walk through array only once

File: apps/condo/domains/acquiring/utils/billingFridge.spec.js [hunk 5]
```diff
@@ -0,0 +1,138 @@
...
+
+        test('should contains accountNumber', () => {
+            expect(frozenReceipt).toHaveProperty(['data', 'account', 'number'])
+        })
+
+        test('should provide information about receipt', () => {
+            expect(frozenReceipt).toHaveProperty('data')
+            const { data } = frozenReceipt
+
+            expect(data).toBeDefined()
```
Comment (dkoviazin) [inline apps/condo/domains/acquiring/utils/billingFridge.spec.js:46]: Maybe, work with arrays of keys?

**Review by dkoviazin — APPROVED** [review]

File: apps/condo/domains/acquiring/schema/MultiPayment.js (hunk 4 above)
Comment (AleX83Xpert) [reply apps/condo/domains/acquiring/schema/MultiPayment.js]: Refactored

**Pull request merged**